ranking/opts.go

```diff
@@ -57,10 +57,14 @@ func boolFromQuery(query url.Values, name string) bool {
 }
 
 func addSuffixesForFiletype(suffixes *map[string]float32, filetype string) {
+	// what exactly happens here? what does the number assign do?
+	// i'm just trying to add support for Objective-C source files
```

Comment:
The pre-ranking (happens before actually looking inside the files) will favor files of the specified suffixes, effectively making them more likely to show up in the search results.

Please remove the comment so that I can merge the pull request :).